doc/ref/chap-osh-assign.md

```diff
@@ -168,6 +162,11 @@ These rules are summarized in the following table.
   - `-A`
   - the original BashAssoc
   - <!-- empty -->
+- tr
+  - (others)
+  - <!-- empty -->
+  - N/A
+  - Error
 
 </table>
 
```

Comment:
> When the result is BashAssoc, see [sh-array](#sh-array).

Should be `sh-assoc`

Reply:
Right, thanks. I fixed it in commit 28359e9fc.